pages/linux/blastn.md

```diff
@@ -0,0 +1,28 @@
+# blastn
+
+> Nucleotide-Nucleotide BLAST.
+> More information: <https://www.ncbi.nlm.nih.gov/books/NBK279696/>.
+
+- Print usage and description (ignoring all other parameters):
+
+`blastn -h`
+
+- Align two or more sequences using megablast (default), with the e-value threshold of 1e-9, pairwise output format (default):
```

Comment:
which is `megablast`?

Reply:
megablast is the default algorithm for blastn when not specified. The algorithm could be specified using `-task` option. eg. `-task megablastn`, `-task blastn`

Reply:
blastn and megablastn are two different algorithms.